src/explore-education-statistics-admin/src/pages/publication/__tests__/PublicationMethodologyPage.test.tsx

```diff
@@ -0,0 +1,1221 @@
+import PublicationMethodologyPage from '@admin/pages/publication/PublicationMethodologyPage';
+import { PublicationContextProvider } from '@admin/pages/publication/contexts/PublicationContext';
+import _methodologyService, {
+  BasicMethodologyVersion,
+} from '@admin/services/methodologyService';
+import _publicationService, {
+  ExternalMethodology,
+  MyPublication,
+  MyPublicationMethodology,
+  PublicationContactDetails,
+  UpdatePublicationRequest,
+} from '@admin/services/publicationService';
+import { render, screen, within, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { MemoryRouter, Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import noop from 'lodash/noop';
+import produce from 'immer';
+
+jest.mock('@admin/services/methodologyService');
+jest.mock('@admin/services/publicationService');
+
+const methodologyService = _methodologyService as jest.Mocked<
+  typeof _methodologyService
+>;
+const publicationService = _publicationService as jest.Mocked<
+  typeof _publicationService
+>;
+
+describe('PublicationMethodologyPage', () => {
+  const testMethodology1: MyPublicationMethodology = {
+    methodology: {
+      amendment: false,
+      id: 'methodology-v1',
+      latestInternalReleaseNote: 'this is the release note',
+      methodologyId: 'methodology-1',
+      published: '2021-06-08T00:00:00',
+      slug: 'methodology-slug-1',
+      status: 'Approved',
+      title: 'Methodology 1',
+      owningPublication: {
+        id: 'publication-1',
+        title: 'Publication 1',
+      },
+      permissions: {
+        canApproveMethodology: false,
+        canUpdateMethodology: false,
+        canDeleteMethodology: false,
+        canMakeAmendmentOfMethodology: false,
+        canMarkMethodologyAsDraft: false,
+      },
+    },
+    permissions: {
+      canDropMethodology: false,
+    },
+    owner: true,
+  };
+  const testMethodology1Draft = produce(testMethodology1, draft => {
+    draft.methodology.status = 'Draft';
+    draft.methodology.permissions = {
+      canApproveMethodology: true,
+      canUpdateMethodology: true,
+      canDeleteMethodology: true,
+      canMakeAmendmentOfMethodology: false,
+      canMarkMethodologyAsDraft: true,
+    };
```

Comment:
We're repeating this set of permissions across all the test methodologies. Could extract it out as a variable to save some lines.